katalog/gatekeeper/core/deploy.yml

```diff
@@ -33,6 +33,8 @@ spec:
         - --operation=audit
         - --operation=status
         - --logtostderr
+        - --health-addr=:9090
+        - --prometheus-port=8888
```

Comment:
Does it makes sense to create a `ServiceMonitor` to monitor the gatekeeper deployment¿? Or maybe was already there and need to be changed the port from where to scrape metrics¿?

Reply:
I say nothing. Everything looks fine here